source/features/pr-filters.tsx

```diff
@@ -27,17 +27,14 @@ function addDropdownItem(dropdown: HTMLElement, title: string, filterCategory: s
 		q: query + (isSelected ? '' : ` ${filterQuery}`)
 	});
 
-	const icon = checkIcon();
-	icon.classList.add('SelectMenu-icon', 'SelectMenu-icon--check');
-
 	dropdown.append(
 		<a
 			href={`?${String(search)}`}
 			className="SelectMenu-item"
 			aria-checked={isSelected ? 'true' : 'false'}
 			role="menuitemradio"
 		>
-			{icon}
+			<CheckIcon className="SelectMenu-icon SelectMenu-icon--check"/>
```

Comment:
Advantage: get icon and add classes in one go (bonus: skip `icon` variable altogether)